src/index.tsx

```diff
@@ -36,6 +36,7 @@ interface Props {
   imageUpload: (file: Blob) => Promise<Record<string, unknown>>;
   plugins?: ((editor: Editor) => Editor)[];
   domParser?: (value: string) => HTMLDocument;
+  theme?: string;
```

Comment:
Since we only use 'dark' explicitly, it might be an idea to either set it to a boolean, e.x `darkmode` and check whether it is true when setting the `data-theme`. Or we should set the type to either 'light' or 'dark' (like an enum).
Personally I'm more fan of the first option.